projects/storefrontlib/cms-components/navigation/scroll-to-top/scroll-to-top.component.spec.ts

```diff
@@ -60,6 +61,16 @@ describe('ScrollToTopComponent', () => {
     el = fixture.debugElement;
   });
 
+  function getKeyboardFocusableElement(): HTMLElement {
```

Comment:
Suggestion: Perhaps add something to the name that will explain more what the function will do (get the top/first focusable element), like `getFirstKeyboardFocusableElement`